[Snyk] Security upgrade eslint from 5.15.0 to 9.0.0

iketheadore opened the pull request:
![snyk-top-banner](https://res.cloudinary.com/snyk/image/upload/r-d/scm-platform/snyk-pull-requests/pr-banner-default.svg)

### Snyk has created this PR to fix 1 vulnerabilities in the npm dependencies of this project.

#### Snyk changed the following file(s):

- `js/package.json`
- `js/package-lock.json`




#### Vulnerabilities that will be fixed with an upgrade:

|  | Issue | Score | 
:-------------------------:|:-------------------------|:-------------------------
![medium severity](https://res.cloudinary.com/snyk/image/upload/w_20,h_20/v1561977819/icon/m.png 'medium severity') | Prototype Pollution <br/>[SNYK-JS-JSYAML-13961110](https://snyk.io/vuln/SNYK-JS-JSYAML-13961110) | &nbsp;&nbsp;**631**&nbsp;&nbsp; 




---

> [!IMPORTANT]
>
> - Check the changes in this PR to ensure they won't cause issues with your project.
> - Max score is 1000. Note that the real score may have changed since the PR was raised.
> - This PR was automatically created by Snyk using the credentials of a real user.

---

**Note:** _You are seeing this because you or someone else with access to this repository has authorized Snyk to open fix PRs._

For more information: <img src="https://api.segment.io/v1/pixel/track?data=eyJ3cml0ZUtleSI6InJyWmxZcEdHY2RyTHZsb0lYd0dUcVg4WkFRTnNCOUEwIiwiYW5vbnltb3VzSWQiOiI1Y2ZmMTdkOS01ODY2LTRhOTktYWM5Zi1jNGJjMzFjZTAwN2QiLCJldmVudCI6IlBSIHZpZXdlZCIsInByb3BlcnRpZXMiOnsicHJJZCI6IjVjZmYxN2Q5LTU4NjYtNGE5OS1hYzlmLWM0YmMzMWNlMDA3ZCJ9fQ==" width="0" height="0"/>
🧐 [View latest project report](https://app.snyk.io/org/gz-c/project/9e744e25-6407-41c3-b4fd-1fa163e2d257?utm_source&#x3D;github&amp;utm_medium&#x3D;referral&amp;page&#x3D;fix-pr)
📜 [Customise PR templates](https://docs.snyk.io/scan-using-snyk/pull-requests/snyk-fix-pull-or-merge-requests/customize-pr-templates?utm_source=github&utm_content=fix-pr-template)
🛠 [Adjust project settings](https://app.snyk.io/org/gz-c/project/9e744e25-6407-41c3-b4fd-1fa163e2d257?utm_source&#x3D;github&amp;utm_medium&#x3D;referral&amp;page&#x3D;fix-pr/settings)
📚 [Read about Snyk's upgrade logic](https://docs.snyk.io/scan-with-snyk/snyk-open-source/manage-vulnerabilities/upgrade-package-versions-to-fix-vulnerabilities?utm_source=github&utm_content=fix-pr-template)

---

**Learn how to fix vulnerabilities with free interactive lessons:**

🦉 [Prototype Pollution](https://learn.snyk.io/lesson/prototype-pollution/?loc&#x3D;fix-pr)

[//]: # 'snyk:metadata:{"breakingChangeRiskLevel":null,"FF_showPullRequestBreakingChanges":false,"FF_showPullRequestBreakingChangesWebSearch":false,"customTemplate":{"variablesUsed":[],"fieldsUsed":[]},"dependencies":[{"name":"eslint","from":"5.15.0","to":"9.0.0"}],"env":"prod","issuesToFix":["SNYK-JS-JSYAML-13961110"],"prId":"5cff17d9-5866-4a99-ac9f-c4bc31ce007d","prPublicId":"5cff17d9-5866-4a99-ac9f-c4bc31ce007d","packageManager":"npm","priorityScoreList":[631],"projectPublicId":"9e744e25-6407-41c3-b4fd-1fa163e2d257","projectUrl":"https://app.snyk.io/org/gz-c/project/9e744e25-6407-41c3-b4fd-1fa163e2d257?utm_source=github&utm_medium=referral&page=fix-pr","prType":"fix","templateFieldSources":{"branchName":"default","commitMessage":"default","description":"default","title":"default"},"templateVariants":["updated-fix-title","priorityScore"],"type":"auto","upgrade":["SNYK-JS-JSYAML-13961110"],"vulns":["SNYK-JS-JSYAML-13961110"],"patch":[],"isBreakingChange":true,"remediationStrategy":"vuln"}'

olemis commented on Makefile:
```diff
@@ -116,11 +117,9 @@ clean-js:
 #----------------
 
-install-deps-nanopb: ## Install tools to generate protobuf classes for C and Python with nanopb
+install-deps-nanopb: /usr/local/bin/protoc ## Install tools to generate protobuf classes for C and Python with nanopb
```
dependent target should be `build-protoc`

snyk-bot committed:
fix: js/package.json & js/package-lock.json to reduce vulnerabilities

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/SNYK-JS-JSYAML-13961110

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/skycoin/hardware-wallet-protob?pullRequest=21) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/skycoin/hardware-wallet-protob?pullRequest=21) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/skycoin/hardware-wallet-protob?pullRequest=21) it.</sub>
